### ISSUE TITLE
CLOUDCMD_TERMINAL environment variable inside Docker container does not function as intended

### ISSUE THREAD

**Issue by ChipWolf** [coderaiser/cloudcmd#283]
* **Version** (`cloudcmd -v`): v14.3.8
* **Node Version** `node -v`:  v13.12.0
* **OS** (`uname -a` on Linux): Linux 35d15636b15d 4.19.76-linuxkit #1 SMP Thu Oct 17 19:31:58 UTC 2019 x86_64 GNU/Linux
* **Browser name/version**: Chromium 80.0.3987.162 (64-bit)
* **Used Command Line Parameters**: 
* **Changed Config**: CLOUDCMD_TERMINAL=false

Using the Docker container, the environment variable `CLOUDCMD_TERMINAL` set to false does not disable the terminal. It must instead be lowercase, `cloudcmd_terminal`, presumably this is because the container default is defined in lowercase here: https://github.com/coderaiser/cloudcmd/blob/master/docker/Dockerfile#L18

**Comment by coderaiser** [comment]
Could you please tell me how do you run `docker container` :)?

**Comment by coderaiser** [comment]
Actually both variants supported :) with help of [server/env.js](https://github.com/coderaiser/cloudcmd/blob/master/server/env.js).

**Comment by ChipWolf** [comment]
@coderaiser see https://github.com/coderaiser/cloudcmd/pull/284#issuecomment-611476657
Also, with Kubernetes; this example disables both the console and the terminal:
```yaml
- image: "coderaiser/cloudcmd:latest"
  env:
    - name: cloudcmd_terminal
      value: "false"
    - name: CLOUDCMD_CONSOLE
      value: "false"
```
This example only disables the console:
```yaml
- image: "coderaiser/cloudcmd:latest"
  env:
    - name: CLOUDCMD_TERMINAL
      value: "false"
    - name: CLOUDCMD_CONSOLE
      value: "false"
```

**Comment by coderaiser** [comment]
I understand what the problem is, fixed with 24ab06c0, thank you :)

**Comment by coderaiser** [comment]
Landed in [v14.3.9](http://github.com/coderaiser/cloudcmd/releases/tag/v14.3.9) 🎉 . Is it works for you?

**Comment by ChipWolf** [comment]
Confirmed working as expected with Docker image tagged `14.3.10-alpine`
